.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "bridge/harmony/protocol"]
+	path = bridge/harmony/protocol
+	url = https://github.com/harmony-development/protocol.git
```

Comment:
Go tooling doesn't have great support for submodules - will this make it harder to build matterbridge? At the moment you can just `go get` it and it'll build and work.

Reply:
got rid of the submodule